images/runtime/commonbase/Dockerfile

```
@@ -1,4 +1,4 @@
-ARG DEBIAN_FLAVOR
+ARG DEBIAN_FLAVOR="bullseye"
```

any reason for adding default here? 
I think we should let it fail if arg is not passed

---

Yes, default value shouldn't be there. Removing it